processmanager/manager.go

```diff
@@ -40,6 +42,12 @@ const (
 
 	// TTL of entries in the LRU cache holding the executables' ELF information.
 	elfInfoCacheTTL = 6 * time.Hour
+
+	// Maximum size of the LRU cache for frames.
+	frameCacheSize = 16384
```

Comment:
Should this be made configurable? Even a small size should prove to be highly effective now that this is caching frames instead of traces. Since the other small LRUs are not configurable by user (like the ELF cache right next to this), I made this also fixed for now.